### DISCUSSION TITLE
Welcome to mCal Discussions!

### DISCUSSION THREAD

**Discussion by theaungmyatmoe** [myanmarcyberyouths/mCal#53] [Announcements]
# 🙌 Welcome to the Nusthell!

We are developing our **mCal** application to provide additional missing features for the Burmese Calendar. We are actively developing the app, so if you encounter any issues, please open an issue or discuss them with other public contributors.

**Comment by phothinmg** [comment]
ဆရာ ဦးရန်နိင်အေးက အတော်လေ့လာပီးလုပ်ခဲ့တာထင်တယ် ကျနော်လည်း ဆရာ့ရဲ့ ceMmDateTime.js နဲ့ Js ကိုဝါသနာအရ လေ့လာဖြစ်ရင်းလေ့ကျင့်ခဲ့ပါတယ်..


အခု Myanmar Calander စတွေ့ကထဲက အားပေးခဲ့ပါတယ်.. အခုလိုလူငယ်တွေဆက်ခံပီး ပိုကောင်းအောင် ဆောင်ရွက်နေတာကိုတွေ့ရတော့ဝမ်းသာမိတယ်.. 

ကျနော်မြန်မာတွေ ကိုယ်ပိုင်ယူနစ်တွေပိုင်ဆိုင်ထားပီးယနေ့အထိအသုံးပြုနေဆဲဖြစ်တာကို ဂုဏ်ယူပါတယ် 
ဒါကိုခေတ်နဲ့အညီ ဖြစ်အောင်ကြိုးစားအားတဲ့သူအားလုံးကိုလေးစားပါတယ်..

English အရေးအသားမကောင်းပဲ နားလည်မှုလွဲမှာစိုးလို့ မြန်မာလိုပဲ ရေးခဲ့တာပါ..

**Reply by theaungmyatmoe** [reply]
ဟုတ်ပါတယ်။ ကို ရန်နိုင်အေး ရဲ့ Alogorithmn ကိုကျွန်တော်တို့ယူသုံးထားပြီးတော့ပိုပြီးကောင်းအောင်ပြန်လုပ်ထားတာမျိုးဖြစ်ပါတယ်။
